Automattic/wp-calypso
# Store Orders: Switch to edit layout for new order screen

This PR deletes the existing, unused order creation components, and adds new components for creation state. The goal here was to get something functional rendering with the edit components, this is just a step towards the final product. With this groundwork we can make smaller PRs for following work.

When you click New Order now, this is the screen you end up on. It looks similar to the edit screen, with a slight difference in the customer info section.

<img width="1172" alt="empty" src="https://user-images.githubusercontent.com/541093/33783551-43c90f0c-dc2b-11e7-9a2d-4f45d09fa89a.png">

Once some products, fees, and addresses are filled in, it looks more like editing an existing order:

<img width="1171" alt="filled-out" src="https://user-images.githubusercontent.com/541093/33783549-43b0d5a4-dc2b-11e7-910d-6e396e88ca5f.png">

There is no activity log component on this page (nothing has happened yet to show up) – this could leave room for us to add a more create-specific cards (See #15679 for discussion)

cc @kellychoffman @jameskoster to let you know this is starting 🙂 

**To test**

- View `/store/order/:site`
- You should see a screen that looks like the edit screen
- Try adding products, fees, change shipping cost – this should update the total cost
- Add addresses, these should display correctly on the page
- **Saving does not currently work!** So just test that interactions are as expected & no errors crop up.
